public_records_portal/prr.py

```diff
@@ -1432,3 +1455,29 @@ def edit_agency_description(request_id, agency_description_text):
     # edit the agency description field of the request
     app.logger.info("Modifying agency description of the request")
     update_obj(attribute='agency_description', val=agency_description_text, obj_type='Request', obj_id=request_id)
+
+def get_contact_info(request_id):
```

Comment:
This won't always be the requester though, right? What happens if a FOIL officer submits it on behalf of a requester.